fe/fe-core/src/main/java/com/starrocks/server/WarehouseManager.java

```diff
@@ -54,11 +54,13 @@ public WarehouseManager() {
     public void initDefaultWarehouse() {
         // gen a default warehouse
         try (LockCloseable lock = new LockCloseable(rwLock.writeLock())) {
-            Warehouse wh = new DefaultWarehouse(DEFAULT_WAREHOUSE_ID,
-                    DEFAULT_WAREHOUSE_NAME, DEFAULT_CLUSTER_ID);
-            nameToWh.put(wh.getName(), wh);
-            idToWh.put(wh.getId(), wh);
-            wh.setExist(true);
+            if (!nameToWh.containsKey(DEFAULT_WAREHOUSE_NAME)) {
```

Comment:
please add more comments in the code here to tell others the reason why this check is here.

Reply:
done